sct.py

```diff
@@ -304,8 +304,10 @@ def clean_resources(ctx, post_behavior, user, test_id, logdir, dry_run, backend)
 @click.option('--get-all-running', is_flag=True, default=False, help='All running resources')
 @sct_option('--test-id', 'test_id', help='test id to filter by')
 @click.option('--verbose', is_flag=True, default=False, help='if enable, will log progress')
+@click.option('-b', '--backend-type', type=click.Choice(["all", "aws", "gce", "eks", "docker", "azure"]),
+              default="all", help="use specific backend")
 @click.pass_context
-def list_resources(ctx, user, test_id, get_all, get_all_running, verbose):
+def list_resources(ctx, user, test_id, get_all, get_all_running, verbose, backend_type):
```

Comment:
Maybe we should consider add a call to this after clean resources, so in cases of keep resources it would show them in the log ?

Reply:
was added to clean resources